app/views/govuk_wrapper.scala.html

```diff
@@ -70,9 +71,14 @@
     }
 
     @mainContentHeader = {
-    @if(contentHeader.isDefined) {
-        @uiLayouts.main_content_header(contentHeader = contentHeader.get)
-    }
+        @if(contentHeader.isDefined) {
+            @uiLayouts.main_content_header(contentHeader = contentHeader.get)
+        }
+        @if(!isBackLinkDisable) {
```

Comment:
Do we ever want the backlink to be disabled? If we don't then we don't need this if statement logic and we can just have the html added here.